harvard-university-of-worcester.csl

```diff
@@ -0,0 +1,376 @@
+      <substitute>
+        <text macro="anon"/>
+      </substitute>
+    </names>
+  </macro>
+  <macro name="author-short">
+    <names variable="author">
+      <name form="short" delimiter=", " and="symbol" delimiter-precedes-last="never"/>
+      <et-al font-style="normal"/>
+      <substitute>
+        <names variable="translator"/>
+        <text macro="anon"/>
+      </substitute>
+    </names>
+  </macro>
+  <macro name="available-from-access">
+    <choose>
+      <if match="any" variable="URL DOI">
+        <group>
+          <text value="Available from: "/>
+          <choose>
+            <if variable="DOI">
+              <text variable="DOI" prefix="doi:"/>
+            </if>
+            <else>
+              <text variable="URL"/>
+            </else>
+          </choose>
+          <group suffix="]" prefix=" [">
+            <text suffix=": " text-case="capitalize-first" term="accessed"/>
+            <date variable="accessed">
+              <date-part name="day" suffix=" "/>
+              <date-part name="month" suffix=" "/>
+              <date-part name="year"/>
+            </date>
+          </group>
+        </group>
+      </if>
+    </choose>
+  </macro>
+  <macro name="italicised-title">
+    <text variable="title" font-style="italic"/>
+  </macro>
+  <macro name="standard-title">
+    <text variable="title"/>
+  </macro>
+  <macro name="publisher-place">
+    <choose>
+      <if type="broadcast" match="none">
+        <text variable="publisher-place"/>
+        <choose>
+          <if match="any" variable="publisher">
+            <text value=","/>
+          </if>
+        </choose>
+        <text variable="publisher" prefix=" "/>
+        <choose>
+          <if match="any" variable="page">
+            <text value=","/>
+          </if>
+          <else>
+            <text value="."/>
+          </else>
+        </choose>
+      </if>
+    </choose>
+  </macro>
+  <macro name="year-date-citation">
+    <choose>
+      <if variable="issued">
+        <date variable="issued">
+          <date-part name="year"/>
+        </date>
+      </if>
+      <else>
+        <text form="short" term="no date"/>
+      </else>
+    </choose>
+  </macro>
+  <macro name="edition">
+    <choose>
+      <if is-numeric="edition">
+        <group delimiter=" ">
+          <number form="ordinal" variable="edition"/>
+          <text form="long" term="edition" strip-periods="true"/>
+        </group>
+      </if>
+      <else>
+        <text variable="edition"/>
+      </else>
+    </choose>
+  </macro>
+  <macro name="pages">
+    <label text-case="lowercase" variable="page" form="short"/>
+    <text variable="page"/>
+  </macro>
+  <macro name="medium-online">
+    <choose>
+      <if match="any" variable="medium">
+        <text variable="medium" prefix="[" suffix="]"/>
+      </if>
+      <else>
+        <choose>
+          <if match="any" variable="DOI URL">
+            <text value="[Online]"/>
```

Comment:
lets do ```<text term="online" text-case="capitalize-first" prefix="[" suffix="]"/>``` Same output, but style is going to be better suited to future modifications.